src/App.js

```diff
@@ -23,25 +24,39 @@ axios.defaults.baseURL = '/';
  *
  * @type {React.FunctionComponent}
  */
-export const App = () => (
-  <Router>
-    <ContextProvider>
-      <Navbar />
-      <div className='container'>
-        <Switch>
-          <Route exact path='/' component={HomePage} />
-          <Route path='/login' component={LoginPage} />
-          <Route path='/signup' component={RegisterPage} />
-          <Route path='/settings' component={SettingPage} />
-          <Route path='/notification' component={NotificationPage} />
-          <Route exact path='/u/:userHandle' component={UserPage} />
-          <Route
-            exact
-            path='/u/:userHandle/post/:postId'
-            component={UserPage}
-          />
-        </Switch>
-      </div>
-    </ContextProvider>
-  </Router>
-);
+export const App = () => {
+  const AppContent = () => (
+    <Router>
+      <ContextProvider>
+        <Navbar />
+        <div className='container'>
+          <Switch>
+            <Route exact path='/' component={HomePage} />
+            <Route path='/login' component={LoginPage} />
+            <Route path='/signup' component={RegisterPage} />
+            <Route path='/settings' component={SettingPage} />
+            <Route path='/notification' component={NotificationPage} />
+            <Route exact path='/u/:userHandle' component={UserPage} />
+            <Route
+              exact
+              path='/u/:userHandle/post/:postId'
+              component={UserPage}
+            />
+          </Switch>
+        </div>
+      </ContextProvider>
+    </Router>
+  );
+  return (
+    <div>
+      <Helmet>
```

Comment:
Can Helmet not just live above Navbar?

Reply:
That's what I tried at first, but for whatever reason, the description wouldn't show up unless it was setup this way. Go figure.

Reply:
Decided to convert it over into a context